db/c_level_db.go

```diff
@@ -49,6 +51,8 @@ func NewCLevelDB(name string, dir string) (*CLevelDB, error) {
 	return database, nil
 }
 
+// Get gets the value for the given key. It returns ErrNotFound if the DB does not contains the
```

Comment:
This doesn't sound round. The function panics on err, and should return nil if the key is not there. Is that not what it does?

Reply:
You are right. That doesn't sound right. I think I might have thought about something else while typing this :-)

Reply:
These comments are not necessary, we should just comment the interface functions (for DB and Iterator)